tifffile/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: tifffile
+  version: "0.4"
+
+source:
+  fn: tifffile-0.4.tar.gz
+  url: https://pypi.python.org/packages/source/t/tifffile/tifffile-0.4.tar.gz
+  md5: d92dba2d7ea08e47bfbf9282c2d2e8dc
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - tifffile = tifffile:main
+    #
+    # Would create an entry point called tifffile that calls tifffile.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
```

Comment:
These requirements aren't right:

- IIUC, `libtiff` is not a dependency of `tifffile` (not even an optional dependency).
- The `tifffile.py` docstring is specific about version requirements:
  - `python 2.7*|>=3.4`
  - `numpy >=1.8.2`
- `matplotlib` is an optional dependency.  I think it should not be listed here. User who need plotting can install it separately.